content/swiftui/concepts/navigation/navigation.md

```diff
@@ -0,0 +1,164 @@
+---
+Title: 'Navigation'
+Description: 'Navigation is the ability to navigate to different views from the root view in the app.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'iOS'
+CatalogContent:
+  - 'learn-iOS-course'
+  - 'path/iOS-path'
+---
+
+**Navigation** is the ability to navigate to different views from the root view in the app. In SwiftUI, `NavigationStack` is used to enable the root view to present views over it.  In iOS 15 and before, `NavigationView` was used, but it's been deprecated since iOS 16 introduced `NavigationStack`. 
+
+## NavigationStack
+
+To be able to navigate to different views from the root view, the view will be warped with `NavigationStack` struct as in the example below: 
+
+```Swift
+struct Tap: View {
+    var body: some View {
+      NavigationStack{
+        Text("Hello, World!")
+       }
+    }
+}
+```
+
+This is the first step to be able to navigate to different views. This defines the root view of the stack. Then to define the destination use `NavigationLink`. When pressed, it navigates to the destination view that was defined: 
```

Comment:
Be a bit definitive and less directional in language. So rather than "This is the first step" -> "To navigate to different views..."